README.md

```diff
@@ -29,126 +29,70 @@ Install [golang](https://golang.org/), [tomlq](https://tomlq.readthedocs.io/en/l
 
 The command will compile and install nolusd locally on your machine.
 
-#### Initialize, set up the DEX parameters and run
+#### Initialize, set up and run
 
-`init-local-network.sh` generates a network setup, including setting up the initial DEX. The Hermes relayer is used to connect to the DEXes, and its configuration is also handled by the script.
+`init-local-network.sh` generates a network setup, including the deployment of platform contracts (only) and initial Hermes setup (Nolus chain configuration).
 
 First, generate the mnemonic you will use for Hermes:
 
 ```sh
 nolusd keys mnemonic
 ```
 
-Then recover it on the DEX (the network binary is required) and use a faucet to obtain some amount:
-
-Example for the Osmosis DEX ([Osmo-test-5 faucet](https://faucet.osmotest5.osmosis.zone/)):
-
-```sh
-osmosisd keys add hermes_key --recover
-```
-
 Initialize and start (run `./scripts/init-local-network.sh --help` for additional configuration options):
 
 ```sh
 ./scripts/init-local-network.sh --reserve-tokens <reserve_account_init_tokens> --hermes-mnemonic <the_mnemonic_generated_by_the_previous_steps> --dex-network-addr-rpc-host <dex_network_addr_rpc_host_part> --dex-network-addr-grpc-host <dex_network_addr_grpc_host_part> --dex-admin-mnemonic <mnemonic_phrase> --store-code-privileged-account-mnemonic <mnemonic_phrase>
```

Comment:
dex-network-addr-rpc-host,
dex-network-addr-grpc-host
probably should be removed here as well